contrib/storage-hive/hive-exec-shade/pom.xml

```diff
@@ -39,23 +39,28 @@
           <groupId>log4j</groupId>
         </exclusion>
         <exclusion>
-          <groupId>commons-codec</groupId>
-          <artifactId>commons-codec</artifactId>
-        </exclusion>
-        <exclusion>
-          <artifactId>calcite-avatica</artifactId>
-          <groupId>org.apache.calcite</groupId>
+          <groupId>org.json</groupId>
+          <artifactId>json</artifactId>
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.apache.parquet</groupId>
+      <artifactId>parquet-column</artifactId>
+      <version>${parquet.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.tdunning</groupId>
+      <artifactId>json</artifactId>
+    </dependency>
   </dependencies>
 
   <build>
     <plugins>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-shade-plugin</artifactId>
-        <version>2.1</version>
+        <version>3.1.0</version>
```

Comment:
What is the reason for the version change and should it be applied to other modules where shading is used?

Reply:
Can it be unified in the plugin management of the Drill root pom?